contrib/docker/cron_pixelfed

```diff
@@ -0,0 +1,10 @@
+# .----------- minute (0 - 59)
+# |     .--------- hour (0 - 23)
+# |     |       .------- day of month (1 - 31)
+# |     |       |       .------- month (1 - 12) OR jan,feb,mar,apr ...
+# |     |       |       |       .---- day of week (0 - 6) (Sunday=0 or 7)  OR sun,mon,tue,wed,thu,fri,sat
+# |     |       |       |       |
+# *     *       *       *       *       user           command to be executed
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:
+TERM=xterm
+*/5     *       *       *       *       application    /usr/local/bin/php /app/artisan schedule:run >/dev/null 2>&1
```

Comment:
Where do the cron output go? They need to get piped to the docker stdout so errors can be detected and tracked. 

You can use this for inspiration: https://github.com/intentionally-left-nil/pixelfed-docker/blob/main/config/periodic/task.sh

Reply:
Good catch, I'll update the cronjob to log to stdout as well.